openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -56,6 +59,12 @@ <h3 class="section-title">${ section['display_name'] }</h3>
                 scored = 'scored' if subsection.get('scored', False) else ''
                 graded = 'graded' if subsection.get('graded') else ''
                 %>
+                % if not subsection.get('complete', True) and subsection.get('due', timezone.now() + timedelta(1)) < timezone.now() and (not reset_deadlines_banner_displayed):
```

Comment:
Nit: don't need the parens at the end around the `not reset_deadlines_banner_displayed`